app/build.gradle

```diff
@@ -4,8 +4,10 @@ apply plugin: 'jacoco'
 apply plugin: 'com.android.application'
 apply plugin: 'me.tatarka.retrolambda'
 apply plugin: 'kotlin-android'
+apply plugin: 'com.google.firebase.firebase-crash'
```

Comment:
It can upload the mapping file to FirebaseCrash.
But, I fogot to introduce this to deploy process.
I'll fix it tomorrow.